app/pages/euDetails/EuSendGoodsAddressPage.scala

```diff
@@ -16,14 +16,24 @@
 
 package pages.euDetails
 
-import models.{Index, InternationalAddress}
+import controllers.euDetails.{routes => euRoutes}
+import models.{CheckMode, Index, InternationalAddress, NormalMode, UserAnswers}
 import pages.QuestionPage
 import play.api.libs.json.JsPath
+import play.api.mvc.Call
 
 case class EuSendGoodsAddressPage(index: Index) extends QuestionPage[InternationalAddress] {
 
   override def path: JsPath = JsPath \ "euDetails" \ index.position \ toString
 
   override def toString: String = "euSendGoodsAddress"
 
+  override protected def navigateInNormalMode(answers: UserAnswers): Call =
+    euRoutes.CheckEuDetailsAnswersController.onPageLoad(NormalMode, index) //TODO navigate to trading address page
```

Comment:
What's stopping the TODOs from being completed?

Reply:
completed now